ProtocolConformance.md

```diff
@@ -1,146 +1,294 @@
-raised at compile time).[^1]
-
-[^1]: Given the way conditional declarations work or don’t work, I’m not sure
-      these declared-on-same-type situations could arise in a meaningful
-      way. Thoughts?
-
-## 1.5 Set of Protocol Witnesses
-
-Given a declaration that a type conforms to a protocol, the protocol witness for
-the conformance is the set consisting of the protocol witness for each declared
-requirement of the protocol.  Inherited requirements of a protocol are
-irrelevant to determination of a protocol witness set.
-
-There is only one protocol witness set for a protocol conformance declaration.
-Such set is immutable, and is not subject to replacement.
-
-If a protocol has no declared requirements, the protocol witness set for
-conformances to the protocol is empty.
+## 1.1 Creation of a Conformance
+A **protocol conformance** is the set of witnesses used by a concrete type to satisfy the requirements of a 
+protocol.  The syntactic declaration, `T: P`, that type `T` conforms to protocol `P` is separate and distinct 
+from the determination of the set of witnesses that is a conformance.  The substance of a conformance is not 
+declared.  It is created in response to a type being declared to conform to a protocol.  The set of witnesses 
+that is a conformance is inferred from the context.
+
+For a type `T` to conform to a protocol `P`, `T` must be declared to conform to `P`, and `T` must have at 
+least one implementation for each protocol requirement of `P`.  A named concrete type is declared to conform to 
+a protocol in one of two ways.  A non-generic concrete type is expressly declared to conform to a protocol.  A 
+concretization of a generic type is implicitly declared to conform to a protocol based on the pattern established 
+by the generic type's declaration and extensions.
+
+><sub>***Conretization of a Generic Type**
+>A concretization of a generic type is a specialized version of the generic type formed by replacing the generic 
+ type's type parameters with concrete type arguments.  A generic type may specify that its concretizations will 
+ conform, or conditionally may conform, to one or more protocols, which establishes a pattern for protocol 
+ conformance.  When a concretization is formed, its protocol conformances are determined by that pattern.*</sub>
+
+A declaration that `T` conforms to `P` further constitutes, with respect to each protocol `O`*<sub>i</sub>* which 
+`P` refines, a declaration that `T` conforms to `O`*<sub>i</sub>*.  Thus, for the declaration of `T: P` to be 
+valid, the implicit declarations of each `T: O`*<sub>i</sub>* must be valid.  For each conformance `T: O`*<sub>i</sub>*, 
+a distinct set of protocol witnesses is established. 
+
+A type `T` cannot be declared to conform to a protocol `P` if, within the visible scope, another declaration 
+exists of `T: P`.  This rule holds true even where competing declarations are conditional with disjoint conditions.  
```

Comment:
Well, that's only true if it's another *explicit* declaration of conformance.

The way to say this is:

```suggestion
If, in a given scope, more than one conformance declaration of a given type to a
given protocol is visible, the program is ill-formed.  This rule holds true even where
 competing declarations are conditional with disjoint conditions.  
```

Perhaps an example here.